### DISCUSSION TITLE
Feature Request: Facebook watch, games, groups, marketplace tabs

### DISCUSSION THREAD

**Discussion by JeromeGill** [jordwest/news-feed-eradicator#432] [Ideas]
Facebook includes sneaky second news feeds for groups, markets and videos. It would be good to eradicate those feeds too.

**Comment by louisabraham** [comment]
Hey, was going to post this issue. Is this already in the live version?

**Comment by polandkathryn** [comment]
Didn't even realise facebook had games until I installed the NewsFeed Eradicator - now definetly want it blocked! Marketplace is a rabbithole too.
